src/Exception/MissingAppConfigurationException.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace AG\ElasticApmLaravel\Exception;
+
+class MissingAppConfigurationException extends \Exception
```

Comment:
`ElasticApmLaravelException` was created as base exception, but instead we are still extending from `Exception`. Is that intentional?

Reply:
Good catch, just a result of copy/paste. It's fixed for both exception classes.